itou/job_applications/tasks.py

```diff
@@ -67,13 +67,26 @@ def notify_pole_emploi_pass(job_application, job_seeker, mode=POLE_EMPLOI_PASS_A
         )
         log.save()
         return False
+
+    # despite some earlier checks, we keep having invalid encrypted indentifier errors
+    if encrypted_nir == "":
```

Comment:
nit: pylint te dirait de mettre `if  not encrypted_nir` ^^

Reply:
ils n'avaient pas donné l'explication dans un mail, qui est que ne NIR n'existe pas de leur coté donc ils répondent avec un encrypted nir vide ?

Reply:
- Concernant la nomenclature, ca dépend de la doc selon laquelle on se réfère… je suis resté sur ce terme mais en effet ce n’est pas vraiment un NIR.
 - Le problème initial semble être (de ce que je comprends) que la recherche du candidat ne donne rien chez eux, visiblement pour autre chose qu'un nom/prénom incorrect. De ce fait, la recherche renvoit
 ```
 { code_sortie: "autre chose que S001", idNationalDE: "" }
 ```

C’est ce qu’on avait conclu dans un des mail en effet. On est censé gérer ce cas (c’était déjà le cas avant leur mail, j’ai patché au cas où, repatché, ca ne fonctionne toujours pas) donc j’ajoute des logs pour voir si les cas considérés déclenchent un problème lié à cet identifiant vide ou si c’est quelque chose de nouveau.

Au passage [les codes sortie documentés](https://github.com/betagouv/itou/blob/143d09a40cf1a296c3e19fbb50337e24eb3db256/itou/utils/apis/pole_emploi.py#L20-L32) sont dans le code.